test/tests.spec.js

```diff
@@ -1062,6 +1062,64 @@ describe('Datetime', () => {
 
 	});
 
+	describe('onSubtractTime', () => {
```

Comment:
You are testing `onAddTime` here and `onSubtractTime` in the next `describe()`, a mixup :)